src/main/java/org/commcare/formplayer/beans/menus/EntityListResponse.java

```diff
@@ -278,6 +280,27 @@ private static EntityBean processEntity(Detail detail, TreeReference treeReferen
         return ret;
     }
 
+    private static String processAutolaunch(SessionWrapper session) {
+        Vector<Action> actions = getActions(session);
+        String ret = null;
+        int index = 0;
+        for (Action action: actions) {
+            if (action.isAutoLaunching()) {
+                ret = "action " + index;
+            }
+            index = index + 1;
+        }
+        return ret;
+    }
+
+    private static Vector<Action> getActions(SessionWrapper session) {
```

Comment:
This is mildly confusing since it shares the name with a normal getter of the `actions` attribute in line 320.

Reply:
This is called a few times, is it cheap, or worth saving somewhere?

Reply:
Pretty sure it's cheap. I'll rename it.